cmd/goal/account.go

```diff
@@ -539,7 +542,11 @@ var rewardsCmd = &cobra.Command{
 	Run: func(cmd *cobra.Command, args []string) {
 		dataDir := ensureSingleDataDir()
 		client := ensureAlgodClient(dataDir)
-		response, err := client.AccountInformation(accountAddress)
+		accountList := makeAccountsList(dataDir)
+		if accountAddress == "" {
+			accountAddress = accountList.getDefaultAccount()
```

Comment:
I think we need to print the address used if default is used.

Reply:
Since the existing rewards command just outputs the rewards amount and nothing else (unlike account list) I didn't want to alter any existing output behaviors.  People could be parsing it for all I knew.   
My (simple) initial intent was just to make some of the commands behave the same as some of the other account commands that honored account aliases (and defaults) as it seemed an odd omission at the time.
If members of the team/community think it makes sense to alter the output I'd love to change it.
Were you thinking just something simple like what goal account list shows for the default account (*Default as the last column?)

Reply:
> Since the existing rewards command just outputs the rewards amount and nothing else (unlike account list) I didn't want to alter any existing output behaviors. People could be parsing it for all I knew.

This is a good point.